fetch.bs

```diff
@@ -1390,7 +1390,8 @@ to not have to set <a for=/>request</a>'s <a for=request>referrer</a>.
   <a>network error</a> if the request is not made to a same-origin URL.
 
   <dt>"<code>cors</code>"
-  <dd>Makes the request a <a>CORS request</a>. Fetch will return a <a>network error</a> if the
+  <dd>For requests whose <a for=request>response tainting</a> is also "<code>cors</code>", makes the
```

Comment:
"gets set to" seems more accurate since response tainting is not set by those creating the request.

Reply:
> "gets set to" seems more accurate since response tainting is not set by those creating the request.

Made it so